admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -211,6 +227,25 @@ const ApiKeysPage = () => {
 									fullWidth
 								/>
 							</TableCell>
+							<TableCell>
+								<Checkbox
+									checked={txnMode === 0? false : true}
+									onChange={handleChange}
+									inputProps={{ 'aria-label': 'controlled' }}
+								/>
+							</TableCell>
+							<TableCell>
+							<TextField
+									type="number"
+									variant="outlined"
+									color="secondary"
+									label="NO_OF_TRANSACTIONS_IN_A_MONTH"
```

Comment:
Is this upposed to have underscores in it and in all caps?

Reply:
I have followed the same on all text labels just to be consistent on the property and the api body its sending